packages/x-license/src/useLicenseVerifier/useLicenseVerifier.ts

```diff
@@ -31,6 +33,12 @@ export function useLicenseVerifier(
 } {
   const { key: contextKey } = React.useContext(MuiLicenseInfoContext);
   return React.useMemo(() => {
+    if (!LicenseInfo.getHasSetLicenseKeyRun()) {
+      showSetLicenseKeyNotCalledError();
+    }
+    if (!LicenseInfo.getIsLicenseKeySet()) {
+      showLicenseKeyNotSetError();
+    }
```

Comment:
The error looks out of place, I thought `LICENSE_STATUS` was meant for this.

Reply:
yeah, that's maybe a bit too much ... will remove it!